docs/examples/haproxy/config.toml

```diff
@@ -3,8 +3,9 @@ dockerURL = "unix:///var/run/docker.sock"
 
 [[extensions]]
 name = "haproxy"
-configPath = "/usr/local/etc/haproxy.cfg"
-pidPath = "/etc/haproxy.pid"
+configPath = "/usr/local/etc/haproxy/haproxy.cfg"
```

Comment:
Is this the actual path of the config?  I thought it was at `/usr/local/etc/haproxy.cfg`.